modules/cluster-management/src/main/java/org/apache/ignite/internal/cluster/management/topology/api/LogicalNode.java

```diff
@@ -79,10 +78,9 @@ public Map<String, String> nodeAttributes() {
         return nodeAttributes;
     }
 
-    /** {@inheritDoc} */
-    @Override
-    public String toString() {
-        // TODO: S.toString for inherited classes do not work properly https://issues.apache.org/jira/browse/IGNITE-19183
-        return S.toString(LogicalNode.class, this);
-    }
+    // TODO: S.toString for inherited classes do not work properly https://issues.apache.org/jira/browse/IGNITE-19183
+    //    @Override
+    //    public String toString() {
+    //        return S.toString(LogicalNode.class, this);
+    //    }
```

Comment:
What was the reason for commenting this code? Maybe you forgot to uncomment it, I don't know

Reply:
Please have a look at the ticket in the TODO above. Current implementation simply prints `LogicalNode {}` hiding all important information, so I'd like to use `toString` from `ClusterNode` instead.

Reply:
This is so weird. In Ignite 2, we constantly used `, "super", super.toString()`, right? Why don't we do the same thing? That's a one line fix

Reply:
I didn't know what we used in Ignite 2 =(